modules/nw-ovn-kuberentes-limitations.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
+
+[id="nw-ovn-kubernetes-limitations_{context}"]
+= OVN-Kubernetes limitations
+
+The OVN-Kubernetes default Container Network Interface (CNI) network provider has a limitation that is related to traffic policies.
+The network provider does not support setting the external traffic policy  or internal traffic policy for a Kubernetes service to `local`.
```

Comment:
In this case should we be even more specific and say _`Service` object_?

Reply:
Until I learn that I'm causing trouble, I favor our doc guidelines for [general references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting) without any styling.